lib/types/library/LibraryBuilder.js

```diff
@@ -40,7 +23,7 @@ class LibraryBuilder extends AbstractBuilder {
 		});
 
 		this.addTask("replaceCopyright", () => {
-			const replaceCopyright = tasks.replaceCopyright;
+			const replaceCopyright = taskRepository.getTask("replaceCopyright").task;
```

Comment:
[minor][consistency] consistent style of call chains?

Reply:
Yes